README.md

```diff
@@ -105,19 +106,25 @@ export ECR_CROMWELL_TAG=<some-value>
 export ECR_NEXTFLOW_ACCOUNT_ID=<some-value>
 export ECR_NEXTFLOW_REGION=<some-value>
 export ECR_NEXTFLOW_TAG=<some-value>
-export ECR_WES_ACCOUNT_ID=<some-value>
-export ECR_WES_REGION=<some-value>
-export ECR_WES_TAG=<some-value>
+export ECR_MINIWDL_ACCOUNT_ID=<some-value>
+export ECR_MINIWDL_REGION=<some-value>
+export ECR_MINIWDL_TAG=<some-value>
 ```
 
-These environment variables point to the ECR account, region, and tags of the Cromwell engine and WES adaptor containers respectively
+These environment variables point to the ECR account, region, and tags of the Cromwell ,Nextflow, and MINIWDL engine respectively
```

Comment:
I don't think miniwdl has any casing. Can you change it from all caps?

Reply:
Fixed